<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RayCaster Map Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/swe.qrc">
    <normaloff>:/Main/mainIcon/mainIcon.svg</normaloff>:/Main/mainIcon/mainIcon.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="we::RenderArea" name="DrawArea" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="cursor">
        <cursorShape>CrossCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Map Properties</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="we::TableMap" name="tableMap"/>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Cell Properties</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="we::TableCell" name="tableCell"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>853</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="actionSelect"/>
    <addaction name="actionQuickEdit"/>
   </widget>
   <widget class="QMenu" name="menuTile">
    <property name="title">
     <string>Tile</string>
    </property>
    <widget class="QMenu" name="menuWall">
     <property name="title">
      <string>Wall</string>
     </property>
     <addaction name="actionWallType_1"/>
     <addaction name="actionWallType_1b"/>
     <addaction name="actionWallType_2"/>
     <addaction name="actionWallType_2b"/>
     <addaction name="actionWallType_3"/>
     <addaction name="actionWallType_3b"/>
    </widget>
    <addaction name="actionPassable"/>
    <addaction name="separator"/>
    <addaction name="menuWall"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuMode"/>
   <addaction name="menuTile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="topToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="separator"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_as"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="leftToolBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelect"/>
   <addaction name="actionQuickEdit"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="rightToolBar">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="bottomToolBar">
   <property name="windowTitle">
    <string>toolBar_4</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPassable"/>
   <addaction name="separator"/>
   <addaction name="actionWallType_1"/>
   <addaction name="actionWallType_1b"/>
   <addaction name="actionWallType_2"/>
   <addaction name="actionWallType_2b"/>
   <addaction name="actionWallType_3"/>
   <addaction name="actionWallType_3b"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../../resources/swe.qrc">
     <normaloff>:/fichier/icons/file.png</normaloff>:/fichier/icons/file.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>Create New Map</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../../resources/swe.qrc">
     <normaloff>:/fichier/icons/folder_open.png</normaloff>:/fichier/icons/folder_open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open existing map</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../../resources/swe.qrc">
     <normaloff>:/fichier/icons/save.png</normaloff>:/fichier/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save map</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="../../resources/swe.qrc">
     <normaloff>:/fichier/icons/save-as.png</normaloff>:/fichier/icons/save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="toolTip">
    <string>Save map as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../../resources/swe.qrc">
     <normaloff>:/fichier/icons/shutdown.png</normaloff>:/fichier/icons/shutdown.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit Map Editor</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/swe.qrc">
     <normaloff>:/Mode/icons/Select.svg</normaloff>:/Mode/icons/Select.svg</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="actionQuickEdit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/swe.qrc">
     <normaloff>:/Mode/icons/Edit.svg</normaloff>:/Mode/icons/Edit.svg</iconset>
   </property>
   <property name="text">
    <string>QuickEdit</string>
   </property>
   <property name="toolTip">
    <string>QuickEdit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+â‚¬</string>
   </property>
  </action>
  <action name="actionPassable">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/swe.qrc">
     <normaloff>:/Tile/icons/Passable.svg</normaloff>:/Tile/icons/Passable.svg</iconset>
   </property>
   <property name="text">
    <string>Passable</string>
   </property>
  </action>
  <action name="actionWallType_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/swe.qrc">
     <normaloff>:/Tile/icons/WallType1.svg</normaloff>:/Tile/icons/WallType1.svg</iconset>
   </property>
   <property name="text">
    <string>Type 1</string>
   </property>
   <property name="toolTip">
    <string>Wall Type 1</string>
   </property>
  </action>
  <action name="actionWallType_1b">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/swe.qrc">
     <normaloff>:/Tile/icons/WallType1b.svg</normaloff>:/Tile/icons/WallType1b.svg</iconset>
   </property>
   <property name="text">
    <string>Type 1b</string>
   </property>
   <property name="toolTip">
    <string>Wall Type 1b</string>
   </property>
  </action>
  <action name="actionWallType_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/swe.qrc">
     <normaloff>:/Tile/icons/WallType2.svg</normaloff>:/Tile/icons/WallType2.svg</iconset>
   </property>
   <property name="text">
    <string>Type 2</string>
   </property>
   <property name="toolTip">
    <string>Wall Type 2</string>
   </property>
  </action>
  <action name="actionWallType_2b">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/swe.qrc">
     <normaloff>:/Tile/icons/WallType2b.svg</normaloff>:/Tile/icons/WallType2b.svg</iconset>
   </property>
   <property name="text">
    <string>Type 2b</string>
   </property>
   <property name="toolTip">
    <string>Wall Type 2b</string>
   </property>
  </action>
  <action name="actionWallType_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/swe.qrc">
     <normaloff>:/Tile/icons/WallType3.svg</normaloff>:/Tile/icons/WallType3.svg</iconset>
   </property>
   <property name="text">
    <string>Type 3</string>
   </property>
   <property name="toolTip">
    <string>Wall Type 3</string>
   </property>
  </action>
  <action name="actionWallType_3b">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/swe.qrc">
     <normaloff>:/Tile/icons/WallType3b.svg</normaloff>:/Tile/icons/WallType3b.svg</iconset>
   </property>
   <property name="text">
    <string>Type3b</string>
   </property>
   <property name="toolTip">
    <string>Wall Type3b</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>we::RenderArea</class>
   <extends>QWidget</extends>
   <header>RenderArea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>we::TableMap</class>
   <extends>QTableWidget</extends>
   <header>TableMap.h</header>
  </customwidget>
  <customwidget>
   <class>we::TableCell</class>
   <extends>QTableWidget</extends>
   <header>TableCell.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/swe.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionNew</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actNew()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actQuit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_as</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actSaveAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSelect</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actChangeMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuickEdit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actChangeMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPassable</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actChangePattern()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionWallType_1</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actChangePattern()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionWallType_1b</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actChangePattern()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionWallType_2</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actChangePattern()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionWallType_2b</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actChangePattern()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionWallType_3</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actChangePattern()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionWallType_3b</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actChangePattern()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>actOpen()</slot>
  <slot>actNew()</slot>
  <slot>actSave()</slot>
  <slot>actSaveAs()</slot>
  <slot>actQuit()</slot>
  <slot>actChangeMode()</slot>
  <slot>actChangePattern()</slot>
 </slots>
</ui>
